### PR TITLE
[Feature-8026][Document] Add example and notice about task type DataX

### PR DESCRIPTION
**Pull request by QuakeWang**
close: https://github.com/apache/dolphinscheduler/issues/8026

### CODE REVIEW

**Comment by QuakeWang** [comment]
PTAL @zhongjiajie @Tianqi-Dotes

File: docs/en-us/dev/user_doc/guide/task/datax.md [hunk 1]
```diff
@@ -1,17 +1,64 @@
...
+DataX task type for executing DataX programs. For DataX nodes, the worker will execute `${DATAX_HOME}/bin/datax.py` to analyze the input json file.
+
+## Create Task
+
+- Click Project Management -> Project Name -> Workflow Definition, and click the "Create Workflow" button to enter the DAG editing page.
+- Drag the <img src="/img/tasks/icons/datax.png" width="15"/> from the toolbar to the drawing board.
+
+## Task Parameter
+
+-    **Node name**: The node name in a workflow definition is unique.
```
Comment (zhongjiajie) [inline docs/en-us/dev/user_doc/guide/task/datax.md:14]: Please remove the extra indentation
```suggestion
- **Node name**: The node name in a workflow definition is unique.
```

File: docs/en-us/dev/user_doc/guide/task/datax.md [hunk 2]
```diff
@@ -1,17 +1,64 @@
...
+- **Running memory**: the minimum and maximum memory required can be configured to suit the actual production environment.
+- **Predecessor task**: Selecting a predecessor task for the current task will set the selected predecessor task as upstream of the current task.
+
+## Task Example
+
+This example demonstrates importing data from Hive into MySQL.
+
+### Configuring the DataX environment in DolphinScheduler
+
+If you are using the DataX task type in a production environment, it is necessary to configure the required environment first. The configuration file is as follows: `/dolphinscheduler/conf/env/dolphinscheduler_env.sh`.
```
Comment (zhongjiajie) [inline docs/en-us/dev/user_doc/guide/task/datax.md:44]: 👍 Good catch, I think we forget this in previous docs

Comment (QuakeWang) [reply docs/en-us/dev/user_doc/guide/task/datax.md:43]: > 👍 Good catch, I think we forget this in previous docs

OK, I will update it by another pr.

Comment (zhongjiajie) [reply docs/en-us/dev/user_doc/guide/task/datax.md:43]: No, in this time, we should only change `dev` docs, previous content just let it go.

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks

**Pull request merged**